tests/unit/API/Management/ConnectionsMockedTest.php

```diff
@@ -151,8 +155,10 @@ public function testThatConnectionsGetAllPaginates()
 
         $this->assertEquals( 'GET', $api->getHistoryMethod() );
         $this->assertStringStartsWith( 'https://api.test.local/api/v2/connections', $api->getHistoryUrl() );
-        $this->assertContains( 'page=0', $api->getHistoryQuery() );
-        $this->assertContains( 'per_page=5', $api->getHistoryQuery() );
+
+        $query = $api->getHistoryQuery();
+        $this->assertStringContainsString( 'page=0', $query );
```

Comment:
That's a good suggestion, but I'd like to avoid getting refactoring tests in this PR. I was just aiming to get these working with minimum viable changes for the purposes of getting PHP 8 support shipped right away. I'll get a ticket going to improve this though, thank you.